docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -57,7 +57,11 @@ xpack.task_manager.monitored_task_execution_thresholds:
 
 The health API is best consumed by via the `/api/task_manager/_health` endpoint.
 
-Additionally, the metrics are logged in the {kib} `DEBUG` logger at a regular cadence.
+Additionally, there are two ways to consume these metrics:
+
+==== Debug logging
```

Comment:
This is a larger header size than what looks like it's parent header (`Consuming health stats`). Maybe reduce header size?

Reply:
Thanks for pointing that out! I actually tried to make it the next header level but the build process can't erroring out with: 

```
INFO:build_docs:asciidoctor: WARNING: user/production-considerations/task-manager-health-monitoring.asciidoc: line 62: section title out of sequence: expected level 3, got level 4
INFO:build_docs:asciidoctor: WARNING: user/production-considerations/task-manager-health-monitoring.asciidoc: line 78: section title out of sequence: expected level 3, got level 4

    at checkExecSyncError (child_process.js:643:11)
    at execFileSync (child_process.js:661:15)
```

Reply:
Maybe just use bolding with asterisks? Looks like that's how it is on the troubleshooting page.

Reply:
Nice, that's a good idea!

Reply:
You can also fix it by putting [float] above the heading.

[float]
==== Debug logging

Reply:
This sentence needs to be finished:

Additionally, there are two ways to consume these metrics: debug logging and audit logging.